aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -210,14 +216,31 @@ async fn main() {
             }
 
             // Print Activated Features
-            let features = fetch_config::<Features>(&client).unwrap();
+            let features = fetch_config::<Features>(&client)?;
             println!(
                 "Features\n{}",
                 serde_yaml::to_string(
                     &aptos_release_builder::components::feature_flags::Features::from(&features)
-                )
-                .unwrap()
+                )?
             );
+            Ok(())
+        },
+        Commands::PrintPackageMetadata {
+            endpoint,
+            package_address,
+            package_name,
+        } => {
+            let client = aptos_rest_client::Client::new(endpoint);
+            let address = AccountAddress::from_str_strict(&package_address)?;
+            let packages = client
+                .get_account_resource_bcs::<PackageRegistry>(address, "0x1::code::PackageRegistry")
+                .await?;
+            for package in packages.into_inner().packages {
+                if package.name == package_name {
+                    println!("PackageMetadata:\n{}", package);
```

Comment:
Can we add an option to produce json instead of pretty printing (looks like package metadata already supports serialize / deserialize)

Reply:
done